Scripts/install.sh

```diff
@@ -1,11 +1,26 @@
 #!/bin/sh
 
+set -euo pipefail
+
 DOWNLOAD_URI=https://github.com/supermarin/Alcatraz/releases/download/1.1.10/Alcatraz.tar.gz
 PLUGINS_DIR="${HOME}/Library/Application Support/Developer/Shared/Xcode/Plug-ins"
+XCODE_VERSION="$(xcrun xcodebuild -version | head -n1 | awk '{ print $2 }')"
+PLIST_PLUGINS_KEY="DVTPlugInManagerNonApplePlugIns-Xcode-${XCODE_VERSION}"
+BUNDLE_ID="com.mneorr.Alcatraz"
+TMP_FILE="/tmp/${BUNDLE_ID}.xcode-defaults"
```

Comment:
@kattrali not sure `random` is installed on OSX by default.

`mktemp` might work